packages/playground/blueprints/src/lib/steps/activate-plugin.ts

```diff
@@ -68,7 +68,7 @@ export const activatePlugin: StepHandler<ActivatePluginStep> = async (
 				}
 			}
 
-			if ( is_wp_error( $response ) ) {
+			if ( is_wp_error( $response ) || is_null( $response ) ) {
```

Comment:
well, now we can't call `$response->get_error_message()` :p

Reply:
Should be good now.